main.css

```diff
@@ -275,16 +275,29 @@ body {
 	font-family: Menlo;
 }
 
-#active-color-preview {
+#active-color-preview-1 {
 	position: absolute;
-	width: var(--palette-width);
+	width: calc(var(--palette-width)/2.4);
 	height: var(--toolbar-height);
 	border: 1px solid #eef;
 	left: var(--toolbar-width);
 	border-radius: var(--border-radius);
 	margin-left: var(--palette-margin-right);
 	transform: translate(-2px, -1px);
 }
+#active-color-preview-2 {
```

Comment:
There is a lot of overlap between `#active-color-preview-1` and `#active-color-preview-2`.

For the sake of reducing repeated code, can you define them like this?

```css

#active-color-preview-1 #active-color-preview-1 {
...
}

/* Then you can specify the things that are different below */
#active-color-preview-1 {
...
}
#active-color-preview-2 {
...
}